types/ace-modules.d.ts

```diff
@@ -422,6 +422,34 @@ declare module "ace-code/src/layer/gutter" {
         getShowLineNumbers(): boolean;
         setShowFoldWidgets(show?: boolean): void;
         getShowFoldWidgets(): boolean;
+        /**
+         * Hides the fold widget/icon from a specific row in the gutter
+         * @param {number} row The row number from which to hide the fold icon
+         */
+        hideFoldWidget(row: number): void;
+        /**
+         * Shows the fold widget/icon from a specific row in the gutter
+         * @param {number} row The row number from which to show the fold icon
+         */
+        showFoldWidget(row: number): void;
```

Comment:
Should we mark these methods as internal so that they don't show up in our public `.d.ts` files? Or do we see a clear use case for customers using these method?